src/Controller/Admin/DatabaseLogController.php

```diff
@@ -46,7 +46,6 @@ public function index() {
 			->select(['summary'])
 			->where(['type' => 'error'])
 			->group('summary')
-			->orderDesc('id')
```

Comment:
Could we add the summary into the select instead? I am afraid without order it could also return wrong order right?

Reply:
Or how can we rewrite this but not losing the order (latest summaries)?

Reply:
`MAX(id)`

Reply:
@slamkajs What do you think, should we try this?

Reply:
Sorry for the long delay.  When I tested the orderDesc('MAX(id)') I was able to resolve my problem.  This seems to be a solid solution.

Thank you

Reply:
Can you modify your PR?

Reply:
The file has been modified.